build-instruction.md

```diff
@@ -452,7 +452,8 @@ The software was validated on:
 
 ### Software Requirements
 
-- [CMake]\* 3.11 or higher
+- [CMake]\* 3.13 or higher for the distribution package 
+- [CMake]\* 3.10 or higher for the samples and demos only
 - Clang\* compiler from Xcode\* 10.1 or higher
 - Python\* 3.5 or higher for the Inference Engine Python API wrapper
 
```

Comment:
Consider adding the following note about CMake version required for Intel Distribution:

> **NOTE**: Building samples and demos from the Intel® Distribution of OpenVINO™ toolkit package requires CMake\* 3.10 or higher.

Reply:
Added